From fe27daa9c6d44e0dbb7f6d0a823b03cd9c7af1d3 Mon Sep 17 00:00:00 2001
From: sub77 <sub77@ymail.com>
Date: Wed, 7 Sep 2016 03:12:34 +0200
Subject: [PATCH 1/2] Add a hook to add extra packages

This commit provides a way of customizing Dirty Deeds (unofficial) builds
without the need to maintain a fork of vendor_du.

As an example, https://github.com/Thrilleratplay/vendor_extra, I am
changing a few default settings, adding SlimLauncher and replacing
SuperSu with Superuser.

Change-Id: I1308aa688dfae01547f693444a0cf3169de1f627
---
 config/common.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/config/common.mk b/config/common.mk
index 9d35445..ac38130 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -330,6 +330,9 @@ PRODUCT_PROPERTY_OVERRIDES += \
 
 -include vendor/cm-priv/keys/keys.mk
 
+# easy way to extend to add more packages
+-include vendor/extra/product.mk
+
 CM_DISPLAY_VERSION := $(CM_VERSION)
 
 ifneq ($(PRODUCT_DEFAULT_DEV_CERTIFICATE),)
-- 
2.9.3

